src/components/AdminSettings/CustomFonts.vue

```diff
@@ -0,0 +1,199 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<NcSettingsSection :name="t('richdocuments', 'Custom Fonts')"
```

Comment:
```suggestion
	<NcSettingsSection :name="t('richdocuments', 'Custom fonts')"
```

Detail: Should be sentence case, like below for "Available fonts"